src/argilla/client/feedback/dataset.py

```diff
@@ -605,6 +607,7 @@ def delete_dataset(dataset_id: UUID) -> None:
                 )
                 return
             self.argilla_id = argilla_id
+            return f"{rg.active_client().http_client.base_url}/{argilla_id}/annotation-mode"
```

Comment:
I think we should avoid calling the `rg.active_client()` more than once within the same function, so feel free to define a variable on top of this function named `client = rg.active_client()` and then re-use `client` e.g. this line should look like

```suggestion
            return f"{client.http_client.base_url}/{self.argilla_id}/annotation-mode"
```